src/binance.mjs

```diff
@@ -116,9 +121,22 @@ export class BinanceAccess {
         this.accounts = new Map();
     }
 
+    async isAvailable() {
+        //Documenation is weak on whether this returns anything other than '{}'.
+        //Monitor this response to see if we get anything else.
+        console.log(this.send(BinanceCommands.available));
```

Comment:
See later comment on send - but calling send without using the apiCommand function means we're not logging the weight of the request and not handling errors properly.

Reply:
Where are you tracking weight at the moment and whats the point in tracking weight?